<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>484</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>864</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="square">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>302</width>
      <height>302</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>29</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <row>
     <property name="text">
      <string>1s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14s</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15s</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>15</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>1</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="letters">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>142</width>
      <height>22</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>39</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="endMove">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить ход</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelMove">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить ход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Заменить фишки</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="rubbish">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>52</width>
      <height>52</height>
     </rect>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="end_hide">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить сброс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Перетащите сюда сбрасываемые фишки</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newGame">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Новая игра</string>
    </property>
   </widget>
   <widget class="QComboBox" name="numPlayers">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>на</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>игрока</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="scores">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Новая строка</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
